book/07-git-tools/git-credential-read-only

```diff
@@ -15,7 +15,7 @@ exit(0) unless File.exists? path
 
 known = {} # <3>
 while line = STDIN.gets
-    break if line.strip == ''
+    break if line.strip == `"
```

Comment:
```suggestion
    break if line.strip == ``
```
I don't think we should be migrating this code?

Reply:
True. My script is blind to the context and has also applied to code blocks. How does it render?

Reply:
It renders OK, in the HTML output:

![rendering_of_changed_code](https://user-images.githubusercontent.com/34918129/95023223-1a300c80-067c-11eb-8ccf-3624bdd77fe7.png)

I still think we should change it back to what it was previously. I don't know if '" is even valid to use with Ruby code.

Reply:
This isn't correct. The hunk must be completely reverted to the original line.